cmd/quicksilverd/root.go

```diff
@@ -134,8 +135,8 @@ func initTendermintConfig() *tmcfg.Config {
 	cfg := tmcfg.DefaultConfig()
 
 	// peers
-	cfg.P2P.MaxNumInboundPeers = 200
-	cfg.P2P.MaxNumOutboundPeers = 40
+	cfg.P2P.MaxNumInboundPeers = 30
```

Comment:
generally would go higher and 8:1 ratio, but this works and will be much better than the defaults -- when it says it puts a strain on RAM it means ~8MB